[2024-04-03T21:26:18.362+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: IMDB-DAG.download-dataset-tasks.download_title.ratings_task manual__2024-04-03T21:26:15.376773+00:00 [queued]>
[2024-04-03T21:26:18.391+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: IMDB-DAG.download-dataset-tasks.download_title.ratings_task manual__2024-04-03T21:26:15.376773+00:00 [queued]>
[2024-04-03T21:26:18.393+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-04-03T21:26:18.430+0000] {taskinstance.py:2217} INFO - Executing <Task(BashOperator): download-dataset-tasks.download_title.ratings_task> on 2024-04-03 21:26:15.376773+00:00
[2024-04-03T21:26:18.449+0000] {standard_task_runner.py:60} INFO - Started process 3676 to run task
[2024-04-03T21:26:18.493+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'IMDB-DAG', 'download-dataset-tasks.download_title.ratings_task', 'manual__2024-04-03T21:26:15.376773+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/imdb_dab.py', '--cfg-path', '/tmp/tmp83xk0pmy']
[2024-04-03T21:26:18.496+0000] {standard_task_runner.py:88} INFO - Job 36: Subtask download-dataset-tasks.download_title.ratings_task
[2024-04-03T21:26:18.697+0000] {task_command.py:423} INFO - Running <TaskInstance: IMDB-DAG.download-dataset-tasks.download_title.ratings_task manual__2024-04-03T21:26:15.376773+00:00 [running]> on host 092193d62c58
[2024-04-03T21:26:19.137+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='iamraphson' AIRFLOW_CTX_DAG_ID='IMDB-DAG' AIRFLOW_CTX_TASK_ID='download-dataset-tasks.download_title.ratings_task' AIRFLOW_CTX_EXECUTION_DATE='2024-04-03T21:26:15.376773+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-03T21:26:15.376773+00:00'
[2024-04-03T21:26:19.144+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-04-03T21:26:19.154+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'curl -o https://datasets.imdbws.com/title.ratings.tsv.gz -p /opt/***/raws/title.ratings.tsv.gz']
[2024-04-03T21:26:19.356+0000] {subprocess.py:86} INFO - Output:
[2024-04-03T21:26:19.416+0000] {subprocess.py:93} INFO - curl: (3) URL using bad/illegal format or missing URL
[2024-04-03T21:26:19.444+0000] {subprocess.py:97} INFO - Command exited with return code 3
[2024-04-03T21:26:19.478+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 212, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 3.
[2024-04-03T21:26:19.498+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=IMDB-DAG, task_id=download-dataset-tasks.download_title.ratings_task, execution_date=20240403T212615, start_date=20240403T212618, end_date=20240403T212619
[2024-04-03T21:26:19.643+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 36 for task download-dataset-tasks.download_title.ratings_task (Bash command failed. The command returned a non-zero exit code 3.; 3676)
[2024-04-03T21:26:19.682+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-04-03T21:26:19.946+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
